config/consensus.go

```diff
@@ -1192,14 +1196,22 @@ func initConsensusProtocols() {
 
 	Consensus[protocol.ConsensusV34] = v34
 
-	// v33 can be upgraded to v34, with an update delay of 12h:
+	v35 := v34
+	v35.StateProofExcludeTotalWeightWithRewards = true
+
+	v35.ApprovedUpgrades = map[protocol.ConsensusVersion]uint64{}
+
+	Consensus[protocol.ConsensusV35] = v35
+
+	// v33 and v34 can be upgraded to v35, with an update delay of 12h:
 	// 10046 = (12 * 60 * 60 / 4.3)
 	// for the sake of future manual calculations, we'll round that down a bit :
-	v33.ApprovedUpgrades[protocol.ConsensusV34] = 10000
+	v33.ApprovedUpgrades[protocol.ConsensusV35] = 10000
+	v34.ApprovedUpgrades[protocol.ConsensusV35] = 10000
```

Comment:
Both going to the same next version seems ok, I guess?  It might cause some trouble in one of my testing functions - but I suppose that's fixable if it happens.